### PR TITLE
OCA\SystemTags\Activity\Setting unit test

### PR DESCRIPTION
**Pull request by MasterOfDeath**
Signed-off-by: Rinat Gumirov <rinat.gumirov@mail.ru>

### CODE REVIEW

**Review by rullzer — APPROVED** [review]
Nice

File: apps/systemtags/tests/Activity/SettingTest.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+		$this->assertSame('<strong>System tags</strong> for a file have been modified', $this->setting->getName());
+	}
+
+	public function testGetPriority() {
+		$this->assertSame(50, $this->setting->getPriority());
+	}
+
+	public function testCanChangeStream() {
+		$this->assertSame(true, $this->setting->canChangeStream());
```
Comment (ChristophWurst) [inline apps/systemtags/tests/Activity/SettingTest.php]: there's also `assertTrue` :wink:

**Review by ChristophWurst — APPROVED** [review]
Thanks! :v:

File: apps/systemtags/tests/Activity/SettingTest.php (hunk 1 above)
Comment (MasterOfDeath) [reply apps/systemtags/tests/Activity/SettingTest.php]: Ok, thanks.

**Pull request merged**